eo2js-runtime/src/objects/org/eolang/txt/sprintf.js

```diff
@@ -35,19 +38,19 @@ const bytesToHex = function(bts) {
  * }}
  */
 const CONVERSION = {
-  's': function(obj) {
+  's'(obj) {
```

Comment:
@skulidropek looks ugly to me, maybe we can turn it off? Or this is how javascript should look in 2025?